pkg/bff/organization.go

```diff
@@ -56,3 +48,19 @@ func (s *Server) OrganizationFromClaims(c *gin.Context) (org *models.Organizatio
 
 	return org, nil
 }
+
+func (s *Server) OrganizationFromID(id string) (org *models.Organization, err error) {
+	// Organizations are stored by UUID in the database
+	var uuid uuid.UUID
+	if uuid, err = models.ParseOrgID(id); err != nil {
+		log.Error().Err(err).Str("org_id", id).Msg("could not parse organization ID")
```

Comment:
Let's try to push logging to the as close to the endpoint as possible; e.g. most logging should be in the handler; that will make it easier to trace where our logs are.